esphome/components/cse7761/cse7761.cpp

```diff
@@ -217,7 +217,7 @@ void CSE7761Component::get_data_() {
     this->voltage_sensor_->publish_state(voltage);
   }
 
-  for (uint32_t channel = 0; channel < 2; channel++) {
+  for (int channel = 0; channel < 2; channel++) {
```

Comment:
This is going to be 0, 1, or 2. I think it can just be `uint8_t`?
```suggestion
  for (uint8_t channel = 0; channel < 2; channel++) {
```